app/case-studies/page.tsx

```diff
@@ -1,5 +1,47 @@
+import CaseStudyGallery from "../components/CaseStudyGallery";
+import GetInTouchCard from "../components/homepage-components/GetInTouchCard";
+
 const CaseStudies = () => {
-  return <div>Case Studies</div>;
+  return (
+    <main className="flex w-screen flex-col justify-center">
+      <section className="flex w-full flex-col items-center bg-white800 dark:bg-black300">
+        <div className="mt-20 flex max-w-3xl flex-col items-center  px-6 py-12 md:mt-32 md:py-[6.5rem]">
+          <div className="flex flex-col lg:flex-row">
+            <div className="flex">
```

Comment:
It seems you have an unnecessary `<div>` here unless it serves a specific styling or layout purpose that isn't apparent from the code alone.

Reply:
hi @cdsalty this div here is for a specific layout purpose here to make sure the thick orange underline always stays where it needs to